airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/activities/JobCreationAndStatusUpdateActivityImpl.java

```diff
@@ -57,7 +65,14 @@ public AttemptCreationOutput createNewAttempt(final AttemptCreationInput input)
   @Override
   public void jobSuccess(final JobSuccessInput input) {
     try {
+      if (input.getStandardSyncOutput() != null) {
+        final JobOutput jobOutput = new JobOutput().withSync(input.getStandardSyncOutput());
+        jobPersistence.writeOutput(input.getJobId(), input.getAttemptId(), jobOutput);
```

Comment:
@jrhizor we should log a warning here if there is no output and the job is successful right?

Reply:
That I am not sure, It was present in the job submitter so I have backport it. I don't think it should happen but I wonder if this can happen in a specific combination of a source and destination.

I've added a warn.

Reply:
thanks